modules/300-prometheus/hooks/deprecate_trickster_ingresses.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2024 Flant JSC
```

Comment:
```suggestion
Copyright 2025 Flant JSC
```

2025 should be addressed in all new files